src/app/pages/addons/markerWithLabel.css

```diff
@@ -0,0 +1,96 @@
+
+.map-price-container {
```

Comment:
I would like to see if we can give an example without providing custom styles.  The way that these get imported right now, they become global styles on the application and will have conflicts if we start to extend further examples with specific stylesheets that have the same class names.  

It would be nice if we could avoid things like BEM or CSSModules on a library like this where the intent is just to show how to use the API without over complicating the examples.

![image](https://user-images.githubusercontent.com/7675942/28840898-4a8b3bbe-76ad-11e7-8f55-c044bd39ff20.png)